src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -60,20 +60,32 @@ public function load($path/*, ...$paths*/)
     /**
      * Sets values as environment variables (via putenv, $_ENV, and $_SERVER).
      *
-     * Note that existing environment variables are never overridden.
+     * Existing environment variables are never overridden, unless they are listed in the SYMFONY_DOTENV_VARS env var.
```

Comment:
I would not document this feature as this is really just a hack to make the internal PHP web server work. Not a feature we want to promote.